ubuntu/base/dind/Dockerfile

```diff
@@ -0,0 +1,45 @@
+ARG UBUNTU_VERSION=14.04
+ARG DOCKER_VERSION=17.05.0-ce
+FROM teracy/ubuntu-$UBUNTU_VERSION-dind:$DOCKER_VERSION
+
+LABEL authors="hoatle <hoatle@teracy.com>"
```

Comment:
no, this is not mine, please use your name instead.

Reply:
deleted!